### PR TITLE
Add SVGEdit to recommended editors

### PR DESCRIPTION
**Pull request by BDVGitHub**
Add [SVGEdit](https://unpkg.com/svgedit@7.3.4/dist/editor/index.html) to possible design software.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ If you are in doubt, feel free to submit it and we'll have a look.
 
 To be considered popular enough to be within our scope, a brand must be in existence for at least one year (from date of first stable release, where applicable) and meet one of the following metrics of popularity, in order of preference:
 
-1. Its website's Similarweb global rank is in the top 500k.
+1. Its website's [Similarweb global rank](https://www.similarweb.com/top-websites/) is in the top 500k.
```
Comment (LitoMore) [inline CONTRIBUTING.md:89]: This link does not tell the global rank. Would you mind removing the link?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -180,6 +180,7 @@ There are many different tools for editing SVG files, some options include:
 | Name | Description | Platform | Price |
 | :-- | :-- | :-: | :-: |
 | [Inkscape](https://inkscape.org/) | Vector Graphics Editor | Windows, Mac, Linux | Free |
+| [SVGEdit](https://svgedit.netlify.app/editor/index.html) | Vector Graphics Editor | Windows, Mac, Linux | Free (Web) |
```
Comment (LitoMore) [inline CONTRIBUTING.md:183]:
```suggestion
| [SVGEdit](https://svgedit.netlify.app/editor/index.html) | Vector Graphics Editor | Web | Free |
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (BDVGitHub) [reply CONTRIBUTING.md]: Is this the correct link? https://www.similarweb.com/website/ maybe just change?

Comment (LitoMore) [reply CONTRIBUTING.md]: The link is unneeded here.

Comment (BDVGitHub) [reply CONTRIBUTING.md]: Ah ok, should be okay now.

**Review by LitoMore — APPROVED** [review]

**Comment by LitoMore** [comment]
@BDVGitHub Congrats on your first contribution to Simple Icons!

**Pull request merged**